selfhosted-nswag.json

```diff
@@ -3,16 +3,16 @@
   "defaultVariables": null,
   "documentGenerator": {
     "fromDocument": {
-      "url": "lfds-swagger.json",
+      "url": "selfhosted-swagger.json",
       "output": null,
       "newLineBehavior": "Auto"
     }
   },
   "codeGenerators": {
     "openApiToCSharpClient": {
       "classname": "{controller}Client",
-      "namespace": "Laserfiche.Api.Client.Lfds",
-      "output": "src/Lfds/AccessTokensClient.cs",
+      "namespace": "Laserfiche.Api.Client.SelfHosted",
```

Comment:
also need to update this file from SelfHosted -> APIServer

Reply:
done